includes/transformer/class-base.php

```diff
@@ -116,8 +125,74 @@ protected function transform_object_properties( $activity_object ) {
 	 */
 	public function to_object() {
 		$activity_object = new Base_Object();
+		$activity_object = $this->transform_object_properties( $activity_object );
+
+		if ( \is_wp_error( $activity_object ) ) {
+			return $activity_object;
+		}
+
+		$activity_object = $this->set_audience( $activity_object );
 
-		return $this->transform_object_properties( $activity_object );
+		return $activity_object;
+	}
+
+	/**
+	 * Get the content visibility.
+	 *
+	 * @return string The content visibility.
+	 */
+	public function get_content_visibility() {
+		if ( ! $this->content_visibility ) {
+			return ACTIVITYPUB_CONTENT_VISIBILITY_PUBLIC;
+		}
+
+		return $this->content_visibility;
+	}
+
+	/**
+	 * Set the content visibility.
+	 *
+	 * @param string $content_visibility The content visibility.
+	 */
+	public function set_content_visibility( $content_visibility ) {
+		$this->content_visibility = $content_visibility;
+
+		return $this;
+	}
+
+	/**
+	 * Set the audience.
+	 *
+	 * @param Base_Object $activity_object The ActivityPub Object.
+	 *
+	 * @return Base_Object The ActivityPub Object.
+	 */
+	protected function set_audience( $activity_object ) {
+		$public = 'https://www.w3.org/ns/activitystreams#Public';
+		$actor  = Actors::get_by_resource( $this->get_attributed_to() );
+		if ( ! $actor || is_wp_error( $actor ) ) {
+			$followers = array();
+		} else {
+			$followers = $actor->get_followers();
+		}
```

Comment:
```suggestion
		$followers = array();
		if ( method_exists( $actor, 'get_followers' ) ) {
			$followers = $actor->get_followers();
		}
```
Not sure if that's much better?